### ISSUE TITLE
Cause is EventFact instead of String

### ISSUE THREAD

**Issue by michelesalvador** [FamilySearch/gedcom5-java#7]
I can't understand why [`EventFact.caus`](https://github.com/FamilySearch/Gedcom/blob/bf091ee265b411662e0c380810d4d5ada25033b4/src/main/java/org/folg/gedcom/model/EventFact.java#L199) is of class `EventFact` instead of `String`.

In [GEDCOM 5.5](http://user.it.uu.se/~andersa/gedcom/ch2.html#SEVENT_STRUCTURE) Cause is defined as
```
EVENT_STRUCTURE
n  <EVENT_TAG>			{1:1}
...
+1 CAUS <CAUSE_OF_DEATH>	{0:1}
```
And in GEDCOM 5.5.1
```
EVENT_DETAIL
...
n CAUS <CAUSE_OF_EVENT> 	{0:1}
```
Both `<CAUSE_OF_DEATH>` and `<CAUSE_OF_EVENT>` are simple values of `{Size=1:90}`.
No trace of another event nested inside.

Should `EventFact.caus` be of class `String`?

**Comment by michelesalvador** [comment]
There is [a comment in `EventFact`](https://github.com/FamilySearch/Gedcom/blob/bf091ee265b411662e0c380810d4d5ada25033b4/src/main/java/org/folg/gedcom/model/EventFact.java#L27) that suggests the reason why `caus` tag is of class `EventFact`:
>so it can have source citations

It's a noble intent, but in my opinion it's not GEDCOM standard.

**Comment by stoicflame** [comment]
Agreed. 

I'm happy to merge a pull request.

**Comment by michelesalvador** [comment]
Opened pull request #12.

**Comment by stoicflame** [comment]
Closed with #12.
